### PR TITLE
Handled a rare bug where a parent frag id is missing, leaving a more …

### PR DESCRIPTION
**Pull request by sareeves96**
…permanent fix for a more appropriate time

### CODE REVIEW

File: src/deriver/api.py [hunk 1]
```diff
@@ -502,7 +502,7 @@ def _process_seeds_for_brics(self):
             try:
                 parent = Heritage.get(Heritage.frag_id == user_frag.id).parent
             # todo: actual exception is deriver.lib_read.FragmentDoesNotExist, check if we can except just this case
-            except:
+            except:  # pylint: disable=broad-except
                 continue
```
Comment (atomoton) [inline src/deriver/api.py:506]: log the error

Comment (sareeves96) [reply src/deriver/api.py:506]: derp

File: src/deriver/jensen_mutate.py [hunk 2]
```diff
@@ -121,13 +121,15 @@ def mutate(mol, mutation_rate):
 
         new_mol_trial = rxn.RunReactants((mol,))
```
Comment (atomoton) [inline src/deriver/jensen_mutate.py:123]: add a random.shuffle() to `new_mol_trial`

Comment (atomoton) [reply src/deriver/jensen_mutate.py:123]: before the for loop

**Pull request merged**